### PR TITLE
Continuation comments cleanup

### PR DESCRIPTION
**Pull request by paul1956**
Clean up PROTOTYPE markers in Continuation comments cleanup

### CODE REVIEW

**Comment by jcouv** [comment]
@dotnet/roslyn-ide for review of the IDE side in this community PR. Thanks

File: src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb [hunk 1]
```diff
@@ -2424,15 +2420,11 @@ End Module]]>,
...
     <Fact()>
     Public Sub ParseWarningDirective_LineContinuation4()
-        ' PROTOTYPE LanguageVersion.VisualBasic15_3 should be LanguageVersion.VisualBasic15_5
-        ' PROTOTYPE 15.5 should be 16
-        ' PROTOTYPE 37306 should be value of ERRID.ERR_CommentsAfterLineContinuationNotAvailable1
         Assert.Equal(37306, ERRID.ERR_CommentsAfterLineContinuationNotAvailable1)
-        ' PROTOTYPE "37306" must be set to the string value of ERRID.ERR_CommentsAfterLineContinuationNotAvailable1
         Dim tree = ParseAndVerify(<![CDATA[#Enable Warning bc41007 _ 'Comment]]>,
-                                  New VisualBasicParseOptions(LanguageVersion.VisualBasic15_3),
+                                  New VisualBasicParseOptions(LanguageVersion.VisualBasic15),
```
[the remark this answers is not in the document]
Comment (paul1956) [reply src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb]: @jcouv  I thought about that and but VB is different from C# and defaults to 15 (even when running on 15.5) unless 15.x is explicitly specified. I think I should test both. Opinion?

**Comment by jcouv** [comment]
@dotnet/roslyn-compiler for a second review of compiler change.
@dotnet/roslyn-ide for review of IDE change.
Thanks

**Comment by jcouv** [comment]
@dotnet/roslyn-compiler for a second review of compiler change.
@dotnet/roslyn-ide for review of IDE change.
Thanks

**Comment by paul1956** [comment]
ping

File: src/EditorFeatures/VisualBasicTest/CodeActions/AbstractVisualBasicCodeActionTest.vb [hunk 2]
```diff
@@ -27,11 +27,14 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeRefactorings
             Return input.Replace("\n", vbCrLf)
         End Function
 
-        Protected Overloads Async Function TestAsync(initialMarkup As XElement, expected As XElement, Optional index As Integer = 0) As Threading.Tasks.Task
+        Protected Overloads Async Function TestAsync(initialMarkup As XElement, expected As XElement, Optional index As Integer = 0, Optional parseOptions As ParseOptions = Nothing) As Threading.Tasks.Task
             Dim initialMarkupStr = initialMarkup.ConvertTestSourceTag()
             Dim expectedStr = expected.ConvertTestSourceTag()
-
-            Await MyBase.TestAsync(initialMarkupStr, expectedStr, parseOptions:=_compilationOptions.ParseOptions, compilationOptions:=_compilationOptions, index:=index)
+            If parseOptions = Nothing Then
```
Comment (jasonmalinowski) [inline src/EditorFeatures/VisualBasicTest/CodeActions/AbstractVisualBasicCodeActionTest.vb]: Shouldn't this be parseOptions Is Nothing?

Comment (paul1956) [reply src/EditorFeatures/VisualBasicTest/CodeActions/AbstractVisualBasicCodeActionTest.vb]: It works correctly with "Is" or =, I would be happy to change it. Every test passes Nothing except the one I added so the only reason to change is for consistency.

**Comment by paul1956** [comment]
ping one more signoff needed from dotnet/roslyn-interactive

**Comment by jcouv** [comment]
Why do you think this PR needs a sign-off from roslyn-interactive? I only see compiler and IDE changes.

**Comment by jcouv** [comment]
Merged. Thanks

**Pull request merged**